src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -811,7 +811,7 @@ private uint getMethodAttribsInternal(MethodDesc method)
                 // do a dynamic check instead.
                 if (
                     !HardwareIntrinsicHelpers.IsIsSupportedMethod(method)
-                    || !_compilation.IsHardwareInstrinsicWithRuntimeDeterminedSupport(method))
+                    || !_compilation.IsHardwareIntrinsicWithRuntimeDeterminedSupport(method))
```

Comment:
This method is not defined anywhere and name had a typo. Looks like a leftover code from previous work.

Reply:
This is CoreRT-specific code. cc @MichalStrehovsky

Reply:
Yesh, but it's fine, I'll fix this up when I port this change over. I'm fine with introducing a build break here.